### PR TITLE
Add support for lightd in Docker

### PR DESCRIPTION
**Pull request by sbc64**
Adds support for light nodes by using the environment variable flag of docker.

Examples:

Lightnode + testnet:
```
docker run --name binance --rm -e "NODETYPE=lightnode"  -p 27147:27147 binance/binance-node
```

Lightnode + prod:
```
docker run --name binance --rm -e "BNET=prod" -e "NODETYPE=lightnode"   -p 27147:27147 binance/binance-node
```

Fullnode + testnet:
```
docker run --name binance --rm -v binanace-data-testnet:/opt/bnbchaind -p 26657:26657 binance/binance-node
```

Fullnode + prod:
```
docker run --name binance --rm -e "BNET=prod" -v binanace-data-prod:/opt/bnbchaind -p 27147:27147 binance/binance-node
```

Changes:
- Consolidated entry script into a single script.
- Copy the needed binaries directly into /usr/local/bin

### CODE REVIEW

**Comment by varnav** [comment]
I want to note that repository is too large now, and I reworked my Dockerfile to pull binaries directly instead of git. Please take a look:

https://github.com/varnav/binance-node-docker

**Review by ChanarinThintip — APPROVED** [review]
Chanarin

File: docker/Dockerfile [hunk 1]
```diff
@@ -12,56 +9,44 @@ LABEL License="MIT License"
...
-ARG CLIVER
 ENV BVER=$BVER
 ENV CLIVER=$CLIVER
-ARG NODETYPE=fullnode
-#ARG NODETYPE=lightnode
-ENV BNET=testnet
-#ENV BNET=prod
 ENV BNCHOME=/opt/bnbchaind
 
-COPY --from=builder /node-binary/cli/testnet/${CLIVER}/linux/tbnbcli /node-binary/cli/testnet/${BVER}/linux/
```
Comment (ChanarinThintip) [inline docker/Dockerfile:45]: 0951295444

**Review by ChanarinThintip — COMMENTED** [review]
0951295444 #183

File: docker/Dockerfile [hunk 2]
```diff
@@ -12,56 +9,44 @@ LABEL License="MIT License"
...
-COPY --from=builder /node-binary/cli/prod/${CLIVER}/linux/bnbcli /node-binary/cli/prod/${BVER}/linux/
-COPY --from=builder /node-binary/${NODETYPE}/testnet/${BVER}/linux/bnbchaind /node-binary/fullnode/testnet/${BVER}/linux/
-COPY --from=builder /node-binary/${NODETYPE}/prod/${BVER}/linux/bnbchaind /node-binary/fullnode/prod/${BVER}/linux/
-COPY --from=builder /node-binary/${NODETYPE}/testnet/${BVER}/config/* /node-binary/fullnode/testnet/${BVER}/config/
-COPY --from=builder /node-binary/${NODETYPE}/prod/${BVER}/config/* /node-binary/fullnode/prod/${BVER}/config/
-COPY ./bin/*.sh /usr/local/bin/
+# Copy binaries for cli tool
```
Comment (ChanarinThintip) [inline docker/Dockerfile:27]: 0951295444

**Review by ChanarinThintip — COMMENTED** [review]
0951295444

**Comment by iakisme** [comment]
@sbc64: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by iakisme** [comment]
@sbc64: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by iakisme** [comment]
@sbc64: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by iakisme** [comment]
@sbc64: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by sbc64** [comment]
Closing PR because I'm no longer interested in this for the time being.

**Pull request closed without merging**